view/frontend/templates/js/replacejs.phtml

```diff
@@ -218,6 +218,33 @@ ob_start();
     }());
     ////////////////////////////////////////////////////////////////////////
 
+    ////////////////////////////////////////////////////////////////////////
+    // Wait for an object to be defined and
+    // execute a callback when it becomes available
+    ////////////////////////////////////////////////////////////////////////
+    ! function() {
+        window.whenDefined = function(obj, property, callback) {
+            var prop = '_'+property;
+            if (obj[property]) {
+                callback();
+            } else {
+                Object.defineProperty(obj, property, {
```

Comment:
add comment (calls callback when variable is set)

Reply:
done